### DISCUSSION TITLE
D&D5.5

### DISCUSSION THREAD

**Discussion by odraccir** [BahamutDragon/pcgen#1235] [General]
Hi @BahamutDragon , 
do you think that the new 2024 revision will be incorporated as a new gamemode or a separated dataset?
i'm working on it using an alternate Gamemode, at the moment in my repository there is Barbarian, Bard, Cleric (completed) and Fighter (wip) with the Human.

**Comment by BahamutDragon** [comment]
I believe it should be a distinct gamemode to differentiate the ruleset. To my thinking it would be far easier than maintaining two different versions of the books in the same gamemode.
